cmake/SetupCharmProjections.cmake

```diff
@@ -38,16 +38,6 @@ if (PROJECTIONS)
       CMAKE_CXX_LINK_FLAGS
       "${CMAKE_CXX_LINK_FLAGS} -tracemode projections -tracemode summary"
   )
-  # The values 1000 and 1001 are chosen arbitrary but assume that there are
-  # fewer than 1000 user-defined Actions (which should be a reasonable
-  # assumption).
-  set(
-      CMAKE_CXX_FLAGS
-      "${CMAKE_CXX_FLAGS} \
- -D SPECTRE_CHARM_PROJECTIONS \
- -D SPECTRE_CHARM_NON_ACTION_WALLTIME_EVENT_ID=1000 \
- -D SPECTRE_CHARM_RECEIVE_MAP_DATA_EVENT_ID=1001"
-  )
   if (PROJECTIONS_PAPI_COUNTERS)
```

Comment:
I'm on the fence. It might be easier to revive it if we want to, but I'm not entirely sure we'll need/want to, in which case it's just baggage... Thoughts?

Reply:
I think we should keep it.

Reply:
I have not used them before. So if we leave it in I could play around with it.
Also, as you said if we were to revive it, it will be easier if we already have this in.

Reply:
Okay, that sounds good. My vote is to leave it in then :) I can add some comments as to what the code should look like that uses this. Would that be helpful?

Reply:
Sure, why not.

Reply:
Okay, I added some words to the CMake file. Since it's not a fully-implemented feature I don't think it should go in the actual documentation.